From b60ade3d1153162cea7063f7f79a8c8d9828ceed Mon Sep 17 00:00:00 2001
From: Nam Nguyen <nam.nguyen.xb@rvc.renesas.com>
Date: Wed, 23 Nov 2016 11:05:51 +0700
Subject: [PATCH] Scaling down height of Glmark3D demo because of
 wayland-taskbar

Because the Glmark app will be displayed under the taskbar, so that
if we want the demo display full screen and still display the taskbar,
we have to scale down the height of demo a mount equal  the height of
taskbar (at this moment, it's 32)
---
 Glmask3D_Demo.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Glmask3D_Demo.sh b/Glmask3D_Demo.sh
index 8e240a9..4bf5b0b 100755
--- a/Glmask3D_Demo.sh
+++ b/Glmask3D_Demo.sh
@@ -9,7 +9,7 @@ printf "\n\e[32m[Info]:\e[0m\n" > ${TTY}
 printf "\tRunning Glmask 3D Graphics demo\n" > ${TTY}
 printf "\n\e[31mNOTE: Click on Exit icon to exit from running applications\e[0m\n\n" > ${TTY}
 
-glmark2-es2-wayland -b :show-fps=true -s $D_WIDTH'x'$D_HEIGHT &> /dev/null &
+glmark2-es2-wayland -b :show-fps=true -s $D_WIDTH'x'`expr $D_HEIGHT - 32` &> /dev/null &
 
 sleep 15s
 
-- 
1.9.1

